### DISCUSSION TITLE
Allow hiding the titlebar

### DISCUSSION THREAD

**Discussion by citydelver** [zoidsh/meru#252] [Feature Requests]
Hello. I like that I'm able to hide the menu bar, but that (almost) useless titlebar could really go. Is there a way this could be done? Or is it an Electron limitation?

I do like having my Windows bottom bar visible with clock etc., but I even tried using F11 to see if it went fullscreen like in many browsers (as I could simply switch windows using alt+tab). It didn't do anything though.

**Comment by timche** [comment]
Since you didn't answer me in #241 , I assume your OS is Windows? It's possible to do it and there are plans to get rid of the default title bar and replace it with a custom one to match the design. I can also imagine to add a setting to hide the title bar. Making the app fullscreen in Windows is a missing feature that exists natively on macOS though.

**Comment by citydelver** [comment]
Yes, sorry, I'm on Windows 8.1. Those are good news. Are they planned for
soon?

**Reply by timche** [reply]
There's no date. Eventually soon, maybe at some point.
